README.md

```diff
@@ -104,18 +95,70 @@ The lighthouse report results are automatically printed to the **Deploy log** in
 2:35:17 PM: }
 ```
 
-We currently support the following settings, which are passed directly to Lighthouse:
+## Lighthouse plugin configuration options
+
+### Run Lighthouse for different site paths
+
+To understand how your site's performance compares across different parts of your site, you can specify different site paths where you'd like to run Lighthouse audits.  Updates to `netlify.toml` will take effect for new builds.
+
+### Example to audit a site's root path and the contact page
+
+```
+[[plugins]]
+  package = "@netlify/plugin-lighthouse"
+
+  # Generate a Lighthouse report for the site's root path
+  [[plugins.inputs.audits]]
+  path = ""
+
+  # Generate a Lighthouse report for the contact site path
+  [[plugins.inputs.audits]]
+  path = "contact"
+
+```
+
+### Run Lighthouse for the desktop experience
+
+By default, Lighthouse takes a mobile-first performance testing approach and runs its audits for the mobile device experience.  Updates to `netlify.toml` will take effect for new builds.
+
+You can optionally run Lighthouse audits for the desktop experience by including `preset = "desktop"` in your `netlify.toml` file:
 
 ```
 [[plugins]]
   package = "@netlify/plugin-lighthouse"
 
   [plugins.inputs.settings]
     preset = "desktop" # Optionally run Lighthouse using a desktop configuration
-    locale = "es" # Any Lighthouse-supported locale, used to generate reports in a different language
 ```
 
-## Running Locally
+To return to running Lighthouse audits for the mobile experience, just remove the line `preset = "desktop"`. New builds will run Lighthouse for the mobile experience.
+
+### Generate Lighthouse results in other languages
+
+By default, Lighthouse results are generated in English. To preview Lighthouse results in other languages, include the language code from any Lighthouse-supported locale in your `netlify.toml` file. Updates to `netlify.toml` will take effect for new builds.
+
```

Comment:
You can check out the latest Lighthouse supported locales or language codes [here](https://github.com/GoogleChrome/lighthouse/blob/da3c865d698abc9365fa7bb087a08ce8c89b0a05/types/lhr/settings.d.ts#L9).